.eslintrc.js

```diff
@@ -7,5 +7,9 @@ module.exports = {
         react: {
             version: '16.3',
         },
+        propWrapperFunctions: [
+            'whitelistProps',
+            { property: 'whitelistProps', object: 'propTypes' },
+        ],
```

Comment:
Why is this necessary and will it be required everywhere we want to use the function?

Reply:
Based on the last commit this should also be removed.